packages/react-sdk/src/components/PendingCallPanel/PendingCallPanel.tsx

```diff
@@ -0,0 +1,53 @@
+import { CallingState } from '@stream-io/video-client';
+import {
+  useCall,
+  useCallCallingState,
+  useI18n,
+} from '@stream-io/video-react-bindings';
+import { Avatar } from '../Avatar';
+import { PendingCallControls } from './PendingCallControls';
+
+export const PendingCallPanel = () => {
+  const call = useCall();
+  const callingState = useCallCallingState();
+  const { t } = useI18n();
+
+  if (!call) return null;
+
+  const caller = call.data?.created_by;
```

Comment:
```suggestion
  const metadata = useCallMetadata();
  const caller = metadata?.created_by;
```

Let's make this part reactive, although it probably would never change.